containers/kubernetes/how-to/enable-disable-ssh.mdx

```diff
@@ -0,0 +1,65 @@
+---
+meta:
+  title: How to enable or disable SSH
+  description: This page explains how to enable or disable the SSH service in your nodes
+content:
+  h1: How to enable or disable SSH
+  paragraph: This page explains how to enable or disable the SSH service in your nodes
+tags: kubernetes kapsule-cluster
+dates:
+  validation: 2023-02-23
+  posted: 2023-02-23
+categories:
+  - containers
+---
+
+By default, an SSH server is installed and configured on the nodes of your Kubernetes Kapsule cluster.
+To improve the security of your Kapsule cluster, you can disable this SSH server on your nodes.
+
+You can manage and edit the status of the SSH server of your nodes from the [Scaleway Console](https://console.scaleway.com) or by using the [API](https://developers.scaleway.com/en/).
+Here, we show you how to do it from the console.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule cluster
+  - This feature is available on the nodes of your cluster
+</Message>
+
+## How to know if this feature is available on the nodes of your cluster
```

Comment:
I would move this whole section to the bottom I think, as it's more of kind of additional information rather than the main part of the how-to